tarodb/assets_store.go

```diff
@@ -762,6 +787,54 @@ func (a *AssetStore) QueryAssetBalancesByGroup(ctx context.Context,
 	return balances, nil
 }
 
+// TODO(jhb): godoc
+func (a *AssetStore) FetchAssetsWithGroup(ctx context.Context) (
```

Comment:
Is there an upcoming sub-system that uses something like this as part of an interface? Typically, we'd either just return a typedef'd version of the DB output, _or_ return a new type that's defined in the package that consumes the new interface.